src/Error/ErrorHandlerInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tomaj\NetteApi\Error;
+
+use Tomaj\NetteApi\Authorization\ApiAuthorizationInterface;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+use Throwable;
+
+interface ErrorHandlerInterface
+{
+    /**
+     * @param array<mixed> $params
+     */
+    public function handle(Throwable $error, array $params): JsonApiResponse;
```

Comment:
tu mas error ale v default implementaci je exception. Zjednot to, pri named parametroch je toto neziaduci jav